LiskHQ/lisk-desktop
# Bug fix: validate urls on navigation.

### What was the problem?

This PR resolves #5112

### How was it solved?

- Added allowed domains that can be navigated to

### How was it tested?

- Start up the electron app using this command `yarn build:electron && yarn start`
- Open up the dev console `command + shift + i`
- Execute this snippet on the console `location.href = 'https://lisk.com'`
- It would be observed that this link would be opened in default browser 
- Try again by executing this snippet on the console `location.href = 'https://google.com'`
- it should be observed that this link would not be navigated to.
